src/tseda/datastore.py

```diff
@@ -77,17 +77,19 @@ class IndividualsTable(Viewer):
         description="Select samples based on the sample set ID.",
         options=[],
     )
-    population_from = param.Integer(
-        label="Population ID",
-        default=None,
-        bounds=(0, None),
-        doc=("Reassign individuals with this population ID."),
+    population_from = pn.widgets.IntInput(
```

Comment:
What is the reason to not use param? By default I think it will be converted to an IntInput widget anyway. Is there something the param API lacks here?

Reply:
There were some issues with having to click outside the box after writing before being able to hit enter to submit when using param.Integer,  this was not a problem when using IntInput!